src/test/java/org/sagebionetworks/bridge/exporter/handler/PhoneAppVersionInfoTest.java

```diff
@@ -38,12 +38,12 @@ public void normalCase() {
     @Test
     public void sanitizeFields() {
         String metadataText = "{\n" +
-                "   \"appVersion\":\"Bridge-EX\\t\\t2.0\",\n" +
-                "   \"phoneInfo\":\"Extra\\t\\ttabs\"\n" +
+                "   \"appVersion\":\"<p>Bridge-EX 2.0</p>\",\n" +
+                "   \"phoneInfo\":\"<b>formatting tag</b>\"\n" +
```

Comment:
Do you want to test how it handles tabs and quotes, if quotes are in the final output?

Reply:
No, because string sanitization occurs in another class, and I don't want to tightly couple the tests for this class with the implementation in a separate class.